package.json

```diff
@@ -17,7 +17,8 @@
     "ci:lint": "eslint 'src/**/*.js' -c ./.eslintrc.js",
     "ci:test": "jest --ci",
     "test": "jest",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
+    "prettier": "prettier . --write"
```

Comment:
Do we need this? `npm run lint` does the same?

Reply:
`npm run lint` will run `eslint`, `prettier --write .` runs `prettier` on _all_ files

Reply:
Makes sense, a few lines above, we have

```
"lint": "eslint 'src/**/*.js' --quiet --fix",
```

How about changing that to:

```
"lint": "run-s lint:eslint lint:prettier",
"lint:eslint": "eslint 'src/**/*.js' --quiet --fix",
"lint:prettier": "prettier . --write"
```

I just feel like when having both a `lint` and a `prettier`, that one will be left unused.

Talking about that `eslint` line, that one should be expanded as well. To include the same `js` files as prettier.

Reply:
Yes we can create an `.eslintignore` file and add the same directories ignored by prettier.
We need both because they do different things

Reply:
Yeah, I understand now. I'm not sure where my head was yesterday.

I was only thinking: _"But `eslint` already runs `prettier`"_. Completely leaving out that it only runs prettier for files that can be parsed by eslint. While `prettier .` also affects non-code files.

Thanks for the time you put in all these pull-requests. Much appreciated :+1:

Reply:
Yes that's right 👼